### PR TITLE
seminars: add accessibility info

### PR DESCRIPTION
**Pull request by miguelgrc**
* INSPIR-3566

### CODE REVIEW

File: inspire_schemas/records/seminars.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ properties:
     address:
         $ref: elements/address.json
         title: Location of the seminar
+    captions:
```
Comment (michamos) [inline inspire_schemas/records/seminars.yml:35]: we use adjectives for this kind of stuff if possible:
```suggestion
    captioned:
```

File: inspire_schemas/records/seminars.yml [hunk 2]
```diff
@@ -130,6 +133,12 @@ properties:
             Contains a reference to the record replacing the current one, if it
             is marked as :ref:`deleted`.
         title: Record replacing this one
+    material:
```
Comment (michamos) [inline inspire_schemas/records/seminars.yml:136]: why not
```suggestion
    material_urls:
```
to be consistent with `join_urls`?

File: inspire_schemas/records/seminars.yml [hunk 3]
```diff
@@ -130,6 +133,12 @@ properties:
...
+        items:
+            $ref: elements/url.json
+        title: Links to PDFs or slides.
```
Comment (michamos) [inline inspire_schemas/records/seminars.yml:139]: Slides can be PDFs but don't have to:
```suggestion
        title: Links to related materials such as slides.
```
To me it's not clear if we want only slides here or also potentially other stuff.

File: inspire_schemas/records/seminars.yml [hunk 4]
```diff
@@ -130,6 +133,12 @@ properties:
             Contains a reference to the record replacing the current one, if it
             is marked as :ref:`deleted`.
         title: Record replacing this one
+    material_urls:
+        items:
+            $ref: elements/url.json
+        title: Links to related material such as slides.
+        type: array 
+        uniqueItems: true
```
Comment (michamos) [inline inspire_schemas/records/seminars.yml:141]: don't forget to add this to the integration fixture: https://github.com/inspirehep/inspire-schemas/blob/master/tests/integration/fixtures/seminars_example.json

Comment (miguelgrc) [reply inspire_schemas/records/seminars.yml:141]: I did!

**Pull request merged**